sdk/appconfiguration/app-configuration/recordings/node/authentication/recording_token_authentication_works.js

```diff
@@ -1,11 +1,82 @@
+  'Cache-Control',
+  'max-age=86400, private',
+  'Content-Length',
+  '1651',
+  'Content-Type',
+  'application/json; charset=utf-8',
+  'Strict-Transport-Security',
+  'max-age=31536000; includeSubDomains',
+  'X-Content-Type-Options',
+  'nosniff',
+  'Access-Control-Allow-Origin',
+  '*',
+  'Access-Control-Allow-Methods',
+  'GET, OPTIONS',
+  'P3P',
+  'CP="DSP CUR OTPi IND OTRi ONL FIN"',
+  'x-ms-request-id',
+  '7114c866-7347-43a4-a6d8-ed66bd1d0400',
+  'x-ms-ests-server',
+  '2.1.11654.13 - WUS2 ProdSlices',
+  'Set-Cookie',
+  'fpc=Aoqe_rEj5e9Lm_DHrmLQ8Vs; expires=Thu, 13-May-2021 00:31:16 GMT; path=/; secure; HttpOnly; SameSite=None',
+  'Set-Cookie',
+  'esctx=AQABAAAAAAD--DLA3VO7QrddgJg7WevrnUhMl4iVcD8fBB7MW2DvhffHmMwsGd6Y3M4jJ_EpDthS8E6ZoSLe2f5_gNJjJsmo_Ini4jCtbQ061XvYYMrIqntpFPJhaeLOw_yPnhqdxUzCMob7V8_N0EfwXKvCOcTIdn4nDF9Lxwi4ROPBf6ZqIENjsxZr3Fi6OW8FjaMnniggAA; domain=.login.microsoftonline.com; path=/; secure; HttpOnly; SameSite=None',
+  'Set-Cookie',
+  'x-ms-gateway-slice=estsfd; path=/; secure; samesite=none; httponly',
+  'Set-Cookie',
+  'stsservicecookie=estsfd; path=/; secure; samesite=none; httponly',
+  'Date',
+  'Tue, 13 Apr 2021 00:31:15 GMT'
+]);
+
+nock('https://login.microsoftonline.com:443', {"encodedQueryParams":true})
+  .post('/azuretenantid/oauth2/v2.0/token', "client_id=azure_client_id&scope=https%3A%2F%2Fmyappconfig.azconfig.io%2F.default%20openid%20profile%20offline_access&grant_type=client_credentials&client-request-id=56d9a3ed-8cfd-4a46-aa83-e36a22275c32&client_secret=azure_client_secret")
```

Comment:
Make sure to update this branch based on master. I believe the recorder now has this `client-request-id` fix issue by default, cc: @HarshaNalluru